kuma/settings/common.py

```diff
@@ -1830,3 +1830,15 @@ def get_user_url(user):
 # Kuma doesn't index anything, that's done by the Yari Deployer, but we need
 # to know what the index is called for searching.
 SEARCH_INDEX_NAME = config("SEARCH_INDEX_NAME", default="mdn_docs")
+
+PLUS_PRICE_VARIANTS = json.loads(
+    config(
+        "PLUS_PRICE_VARIANTS_JSON",
+        default=json.dumps(
+            [
+                "$X a month or $XX a year",
+                "$Y a month or $YY a year",
+            ]
+        ),
+    )
+)
```

Comment:
Nit. This works fine of course, but it's our convention to use the `cast` feature of `config` for this kind of thing:
```suggestion
PLUS_PRICE_VARIANTS = config(
    "PLUS_PRICE_VARIANTS_JSON",
    default=json.dumps(
        [
            "$X a month or $XX a year",
            "$Y a month or $YY a year",
        ]
    ),
    cast=json.loads
)
```

Reply:
That's much better!